proxy/core/connection/server.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, host: str, port: int) -> None:
         self._conn: Optional[Union[ssl.SSLSocket, socket.socket]] = None
         self.addr: Tuple[str, int] = (host, int(port))
         self.closed = True
+        self.handshake: Dict(int, bytes) = {}
```

Comment:
Don't think this is a valid type hint, I think only `[ ]` brackets will work.

Reply:
this two statement are not used any more too

self.handshake: Dict(int, bytes) = {}
self.session_id: Optional[bytes] = None